test/model/test_controller.py

```diff
@@ -373,23 +405,41 @@ async def test_provision_smart_start_node_qr_info(controller, uuid4, mock_comman
     assert ack_commands[0] == {
         "command": "controller.provision_smart_start_node",
         "entry": {
-            "version": 0,
+            "version": 1,
             "securityClasses": [0],
-            "dsk": "test",
+            "dsk": "test1",
             "genericDeviceClass": 1,
-            "specificDeviceClass": 1,
-            "installerIconType": 1,
-            "manufacturerId": 1,
-            "productType": 1,
-            "productId": 1,
-            "applicationVersion": "test",
-            "maxInclusionRequestInterval": 1,
-            "uuid": "test",
+            "specificDeviceClass": 2,
+            "installerIconType": 3,
+            "manufacturerId": 4,
+            "productType": 5,
+            "productId": 6,
+            "applicationVersion": "test2",
+            "maxInclusionRequestInterval": 7,
+            "uuid": "test3",
             "supportedProtocols": [0],
         },
         "messageId": uuid4,
     }
 
+    provisioning_entry = controller_pkg.QRProvisioningInformation(
+        QRCodeVersion.S2,
+        [SecurityClass.S2_UNAUTHENTICATED],
+        "test1",
+        1,
+        2,
+        3,
+        4,
+        5,
+        6,
+        "test2",
+        7,
+        "test3",
+        [Protocols.ZWAVE],
+    )
+    with pytest.raises(ValueError):
```

Comment:
It would be good to add a comment above each block that tests a ValueError mentioning what we test for. The ValueError doesn't tell us that by itself.